website/components/CodeDocs.tsx

```diff
@@ -174,29 +175,31 @@ function CodeDocs({ component, readme, changelog }: BaseLayoutProps) {
           changelog={changelog}
         />
       ) : (
-        <DesktopInstall
-          component={component}
-          version={version}
-          changelog={changelog}
-        />
-      )}
+          <DesktopInstall
+            component={component}
+            version={version}
+            changelog={changelog}
+          />
+        )}
       <GridContainer align="flex-start" justify="flex-start">
         <GridItem sm={12} md={12} xl={12}>
           <Tabs className={tabsPadding}>
-            {example && (
+            {example ? (
               <Tab default name="Example" className={mt3}>
                 <Code showLineNumbers language="js">
                   {example}
                 </Code>
               </Tab>
-            )}
-            {outputHTML && (
+            ) : <></>}
```

Comment:
we should eventually update the typing of Tabs to support having `null`, `undefined`, and other "empty" values as children

Reply:
https://jira.mongodb.org/browse/PD-936